### PR TITLE
Add "/api/" to "skipped_uris".

### PR DESCRIPTION
**Pull request by chexxor**
Fixes https://github.com/YunoHost-Apps/homeassistant_ynh/issues/161

## Problem

See https://github.com/YunoHost-Apps/homeassistant_ynh/issues/161

## Solution

Setting this "ignored_uris" setting seems to fix it.

## PR Status

- [x] Code finished and ready to be reviewed/tested
- [x] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

File: scripts/install [hunk 1]
```diff
@@ -53,6 +53,8 @@ ynh_app_setting_set --app=$app --key=final_path --value="$final_path"
 ynh_app_setting_set --app=$app --key=data_path --value="$data_path"
 ynh_app_setting_set --app=$app --key=log_file --value="$log_file"
 ynh_app_setting_set --app=$app --key=path_url --value="$path_url"
+# Allow the web UI to use its own API without SSOWat interfering with it.
+ynh_app_setting_set --app=$app --key=skipped_uris --value="/api/"
```
Comment (ewilly) [inline scripts/install:57]: Is that line useful here ?

File: scripts/install [hunk 2]
```diff
@@ -184,6 +186,7 @@ ynh_systemd_action --service_name=$app --action=restart
 ynh_script_progression --message="Configuring permissions..."
 
 [ $is_public -eq 1 ] && ynh_permission_update --permission="main" --add="visitors"
+ynh_app_setting_set --app=$app --key=skipped_uris --value="/api/"
```
Comment (ewilly) [inline scripts/install:189]: Seem to be the old way and is deprecated. 
I suggest : `ynh_permission_create --permission="api" --label="api" --url="/api" --allowed="visitors" "all_users" --auth_header="false" --show_tile="false" --protected="true"`
What's you opinion ?

**Comment by chexxor** [comment]
I wonder if this change removes authentication from the API completely? That's hard to believe because the API still works, and I presume HomeAssistant wouldn't let an unauthenticated user make API calls to add a device to it.

File: scripts/install (hunk 2 above)
Comment (chexxor) [reply scripts/install:189]: I'll research it and give it a try.
Thanks for the review!

**Comment by tituspijean** [comment]
These will only tell YunoHost to let go of any attempt to interfere with the access to `/api`, it should not prevent Home-Assistant from enforcing its own authentication scheme. ;)

**Comment by ewilly** [comment]
Great work, thanks a lot !

**Pull request merged**